xml/libvirt_configuration_virsh.xml

```diff
@@ -376,7 +514,19 @@ CPU Affinity:   yy
 
   <para>
    The amount of memory allocated for the &vmguest; can also be configured with
-   virsh. It is stored in the <tag>memory</tag> element. Follow these steps:
+   virsh. It is stored in the <tag>memory</tag> element and defines the
+   maximum allocation of memory for the &vmguest; at boot time. The optional
+   <tag>currentMemory</tag> element defines the actual memory allocated to
+   the &vmguest;. <tag>currentMemory</tag> can be less than <tag>memory</tag>,
+   allowing for increasing, or ballooning, the memory while the &vmguest; is
```

Comment:
```suggestion
   allowing for increasing (or <emphasis>ballooning</emphasiss> the memory while the &vmguest; is
```

Reply:
This change doesn't look quite right. Probably just the closing ')' is missing, e.g.
(or <emphasis>ballooning</emphasiss>)
I don't know how to fix it as part of this review process, so will not "Add suggestion to batch" for this one.